### ISSUE TITLE
Add more anime reaction images!

### ISSUE THREAD

**Issue by uzair-ashraf** [uzair-ashraf/anime-reaction-api#5]
Check the contributing section in the readme

**Comment by mgomez08** [comment]
I would like to contribute reactions, is it necessary to open a issues by reaction or can multiple reactions be added?

**Comment by uzair-ashraf** [comment]
@mgomez08 Multiple reactions can be added! Just make sure you post them in the comment for the pull request so they can be checked

**Comment by SoldierCorp** [comment]
Hi @uzair-ashraf 

I just opened the issue #8 but I am not allowed to add the labels you are requesting to it also the `hacktoberfest` label needs to be added or should I point the PR to this issue you opened?

**Comment by Xx-Ashutosh-xX** [comment]
I also would like to contribute on this issue.

**Comment by uzair-ashraf** [comment]
@SoldierCorp You can point the pr to the issue opened, thank you!

**Comment by SoldierCorp** [comment]
Thank you @uzair-ashraf!

We are already using your app in our Slack channel 🙂

**Comment by GriffinMLN** [comment]
@uzair-ashraf I like the bot a lot :D Is there anyway that I can contribute more?
Do you accept anime gifs? Maybe add additional categories or re-distribute/classify some of the current images?

**Comment by uzair-ashraf** [comment]
> @uzair-ashraf I like the bot a lot :D Is there anyway that I can contribute more?
> 
> Do you accept anime gifs? Maybe add additional categories or re-distribute/classify some of the current images?

Yes! Only issue with gifs is that I locally host images, so it can be an issue with space. But go for it!
The only category that is off limits is random, since it's used in some logic with the slack shuffle feature
